sdcm/nemesis.py

```diff
@@ -303,13 +307,15 @@ def _destroy_data_and_restart_scylla(self):
         self.target_node.stop_scylla_server(verify_up=False, verify_down=True)
 
         try:
-            file_for_destroy = self._choose_file_for_destroy(ks_cfs)
-
-            result = self.target_node.remoter.run('sudo rm -f %s' % file_for_destroy)
-            if result.stderr:
-                raise FilesNotCorrupted('Files were not corrupted. CorruptThenRepair nemesis can\'t be run. '
-                                        'Error: {}'.format(result))
-            self.log.debug('Files {} were destroyed'.format(file_for_destroy))
+            # Remove 5 data files
+            for _ in xrange(5):
```

Comment:
why 5 files?

Reply:
Roy asked to remove 5 files, not 1. We agreed about 5